src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DeleteOperationTest.java

```diff
@@ -50,48 +57,63 @@ public class DeleteOperationTest extends AbstractValidatingStargateBridgeTest {
   @Nested
   class Execute {
 
+    private final ColumnDefinitions DELETE_RESULT_COLUMNS =
+        buildColumnDefs(Arrays.asList(TestColumn.ofBoolean("[applied]")));
```

Comment:
I added an overload for `buildColumnDefs` to avoid need to pass `List` so can remove `Arrays.asList(...)` wrapping.